cmd/buckd/docker-compose-dev.yml

```diff
@@ -3,31 +3,40 @@ services:
   buckets:
     build:
       context: ../../
-      dockerfile: ./cmd/buckd/Dockerfile
+      dockerfile: ./cmd/buckd/Dockerfile.dev
+    volumes:
+      - "${REPO_PATH}/buckets:/data/buckets"
     environment:
       - BUCK_LOG_DEBUG=true
-      - BUCK_LOG_FILE
       - BUCK_ADDR_API=/ip4/0.0.0.0/tcp/3006
       - BUCK_ADDR_API_PROXY=/ip4/0.0.0.0/tcp/3007
       - BUCK_ADDR_MONGO_URI=mongodb://mongo:27017
-      - BUCK_ADDR_MONGO_NAME
+      - BUCK_ADDR_MONGO_NAME=buckets
       - BUCK_ADDR_THREADS_HOST=/ip4/0.0.0.0/tcp/4006
-      - BUCK_ADDR_THREADS_MONGO_URI
-      - BUCK_ADDR_THREADS_MONGO_NAME
+      - BUCK_ADDR_THREADS_MONGO_URI=mongodb://mongo:27017
+      - BUCK_ADDR_THREADS_MONGO_NAME=buckets_threads
       - BUCK_ADDR_GATEWAY_HOST=/ip4/0.0.0.0/tcp/8006
       - BUCK_ADDR_GATEWAY_URL
       - BUCK_ADDR_IPFS_API=/dns4/ipfs/tcp/5001
       - BUCK_ADDR_POWERGATE_API
       - BUCK_GATEWAY_SUBDOMAINS
+      - BUCK_DNS_DOMAIN
     ports:
       - "127.0.0.1:3006:3006"
-      - "3007:3007"
+      - "127.0.0.1:3007:3007"
       - "4006:4006"
       - "127.0.0.1:8006:8006"
+      - "127.0.0.1:40000:40000"
+    security_opt:
+      - "seccomp:unconfined"
+    cap_add:
+      - SYS_PTRACE
```

Comment:
Some things needed for remote debugging.

Reply:
Cool.